website/docs/r/snat.html.markdown

```diff
@@ -26,6 +27,7 @@ resource "vcd_snat" "outbound" {
 The following arguments are supported:
 
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the SNAT
+* `network_name` - (Optional *v2.2+*) The name of the organization network on which to apply the SNAT. This will be required field in the next major version.
```

Comment:
Please add a semicolon after Optional and an emphasis to the warning as in DNAT rule description request above.

Reply:
fixed